src/components/PaymentMethod/schemas.ts

```diff
@@ -0,0 +1,24 @@
+import { Schema } from 'effect'
+
+export enum PaymentMethod {
+  Cash = 'CASH',
+  Check = 'CHECK',
+  CreditCard = 'CREDIT_CARD',
+  Ach = 'ACH',
```

Comment:
```suggestion
  ACH = 'ACH',
```
traditionally always capitalized